deps.edn

```diff
@@ -65,10 +65,7 @@
                                              :exclusions  [org.clojure/clojurescript]}
   info.sunng/ring-jetty9-adapter            {:mvn/version "0.18.5"}             ; Drop-in replacement for official Ring Jetty adapter. Supports Jetty 11 webserver.
   instaparse/instaparse                     {:mvn/version "1.4.12"}             ; Make your own parser
-  io.forward/yaml                           {:mvn/version "1.0.11"              ; Clojure wrapper for YAML library SnakeYAML.
-                                             :exclusions  [org.clojure/clojure
-                                                           org.flatland/ordered
-                                                           org.yaml/snakeyaml]}
+  clj-commons/clj-yaml                      {:mvn/version "1.0.26"}             ; Clojure wrapper for YAML library SnakeYAML
```

Comment:
`clj-yaml` doesn't cause problems for our deps anymore 🤞 and uses the same version of SnakeYAML we had previously pinned.